build/Dockerfile

```diff
@@ -198,16 +203,16 @@ RUN \
     /bin/bash -c \
     'set -e &&\
     if [[ "$(dpkg --print-architecture)" == "amd64" ]]; then \
-      curl https://dl.google.com/linux/linux_signing_key.pub | apt-key add - &&\
-      echo "deb http://dl.google.com/linux/chrome/deb/ stable main" | tee /etc/apt/sources.list.d/google-chrome.list &&\
-      apt-get update -qq &&\
-      apt-get upgrade -yqq &&\
-      DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends --allow-unauthenticated google-chrome-stable &&\
-      mv /usr/bin/google-chrome-stable /usr/bin/chromium; \
+    curl https://dl.google.com/linux/linux_signing_key.pub | apt-key add - &&\
```

Comment:
I wonder if we should switch to Chromium for `amd64` 🤔  Nowadays the package is up-to-date with Chrome stable (it wasn't the case before).